tools/gen_operators_yaml.py

```diff
@@ -0,0 +1,592 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import sys
+from typing import List, Optional, Dict, Any
+
+import yaml
+from gen_op_registration_allowlist import (
+    canonical_name,
+    gen_transitive_closure,
+    load_op_dep_graph,
+)
+from torchgen.selective_build.operator import *
+from torchgen.selective_build.selector import merge_kernel_metadata
+
+# Generate YAML file containing the operators used for a specific PyTorch model.
+# ------------------------------------------------------------------------------
+#
+# This binary is responsible for generating the model_operators.yaml file for
+# each model from a pt_operator_library() BUCK macro invocation. See
+# https://fb.quip.com/2oEzAR5MKqbD#TNCACA9Ikmy for details.
```

Comment:
Not accessible.  Is the plan to remove the internal version of this tool?

Reply:
They can be shared